solr/src/main/scala/akka/stream/alpakka/solr/javadsl/SolrSinkSettings.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2016-2018 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.stream.alpakka.solr.javadsl
+
+import akka.stream.alpakka.solr._
+import scaladsl.{SolrSinkSettings => ScalaSolrSinkSettings}
+
+final class SolrSinkSettings(val bufferSize: Int, val retryInterval: Int, val maxRetry: Int, val commitWithin: Int) {
```

Comment:
1. Other modules than Elasticsearch keep their settings classes in the main module package instead of having a Java and Scala variant. I think that's a better solution.
1. For the Sink settings, you might want to consider `SolrUpdateSettings` as they are used for both Flow and Sink.
1. `retryInterval` should be a `FiniteDuration` instead